pydicom/tag.py

```diff
@@ -28,6 +28,7 @@ def tag_in_exception(tag):
     try:
         yield
     except Exception as ex:
+        logger.warning("trouble with", exc_info=ex)
         msg = 'With tag {0} got exception: {1}'.format(tag, str(ex))
```

Comment:
Why not just log the message constructed in the next line?

Reply:
This is left over from trying to track down issue #519 (writing a string with value None). I used this to get the stack trace of where this was occurring.  The message just told me "With Tag Manufacturer got exception: TypeError: object of type 'NoneType' has no len() ".
This line told me where.

I could remove it, or change its level to debug to suppress it more.  Any opinions?

Reply:
Can the extra information logging is teasing out just be added to the existing message?  Seems like it would always be helpful.